src/main/kotlin/net/maizegenetics/phgv2/api/HaplotypeGraph.kt

```diff
@@ -350,12 +350,14 @@ class HaplotypeGraph(hvcfFiles: List<String>) {
     /**
      * Returns a map of hapid -> ReferenceRange
      */
-    fun hapIdToRefRangeMap(): MutableMap<String, ReferenceRange> {
-        //hapIdToRefRangeMap is a map of hapid -> ReferenceRange
-        val hapIdToRefRangeMap = mutableMapOf<String, ReferenceRange>()
+    fun hapIdToRefRangeMap(): MutableMap<String, MutableList<ReferenceRange>> {
```

Comment:
Why Mutable? Can this be Map<String, List<ReferenceRange>>?

Reply:
Good question. I removed the mutables from the return type.